### PR TITLE
Fix issue 56

### LINKED ISSUE #56: [#35] can't get data from backend (controller) to frontend on edit page

### PR DESCRIPTION
**Pull request by shin5446**
## content
- fix bug for event edit action
- add flash message
- add PR　template
## image
image
## issue
fixed #56

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Content
+
+## Image
+
+## Related Issue
+fixed
+
+## notes
```
Comment (shin5446) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: This is for the template for PR. I hope It'll work!

File: src/app/Http/Controllers/EventController.php [hunk 2]
```diff
@@ -50,18 +49,20 @@ public function show(Event $event)
     /**
      * Show the form for editing the specified resource.
      */
-    public function edit(Request $event)
+    public function edit(Event $event)
```
Comment (shin5446) [inline src/app/Http/Controllers/EventController.php:52]: I wrongly DI Request class. My apologies.

File: src/resources/js/Pages/Main/Index.jsx [hunk 3]
```diff
@@ -4,10 +4,30 @@ import Card from "./Card";
...
+                    <div class="bg-blue-100 border border-blue-400 text-blue-700 px-4 py-3 rounded relative text-center" role="alert">
+                        <span class="block sm:inline">{flash.message}</span>
+                        <span class="absolute top-0 bottom-0 right-0 px-4 py-3">
+                            <svg onClick={handleCloseFlash} class="fill-current h-6 w-6 text-blue-500" role="button" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+                                <title>Close</title>
+                                <path d="M14.348 14.849a1.2 1.2 0 0 1-1.697 0L10 11.819l-2.651 3.029a1.2 1.2 0 1 1-1.697-1.697l2.758-3.15-2.759-3.152a1.2 1.2 0 1 1 1.697-1.697L10 8.183l2.651-3.031a1.2 1.2 0 1 1 1.697 1.697l-2.758 3.152 2.758 3.15a1.2 1.2 0 0 1 0 1.698z" />
+                            </svg>
+                        </span>
+                    </div>)}
+            </div>
```
Comment (shin5446) [inline src/resources/js/Pages/Main/Index.jsx:30]: I took this design from [Tailwind](https://v1.tailwindcss.com/components/alerts)
If you don't like this, you can change the design to whatever you want!

File: src/resources/js/Pages/Main/Index.jsx [hunk 4]
```diff
@@ -4,10 +4,30 @@ import Card from "./Card";
 import Header from "@/Components/Header";
 import Footer from "@/Components/Footer";
 
-function Index({ auth, events }) {
+function Index({ auth, events, flash }) {
+    // mock data 2023/06/12
+    auth = "";
+    const [isFlashVisible, setFlashVisible] = React.useState(true);
```
Comment (akaka0039) [inline src/resources/js/Pages/Main/Index.jsx:10]: small thing.
but i prefer 
```
const [isFlashVisible, setFlashVisible] = useState(true);
```
and
```
import React, { useState } from "react";
```

let me fix it next commit

**Review by akaka0039 — APPROVED** [review]

**Pull request merged**